gtk/gtkscrolledwindow.h

```diff
@@ -214,6 +214,17 @@ void           gtk_scrolled_window_set_overlay_scrolling  (GtkScrolledWindow *sc
 GDK_AVAILABLE_IN_3_16
 gboolean       gtk_scrolled_window_get_overlay_scrolling (GtkScrolledWindow   *scrolled_window);
 
+GDK_AVAILABLE_IN_3_22
```

Comment:
I don't think we have these 3.22 macros in our version of GTK. Please change the code so that it does not require them (e.g. by changing them to 3.20 instead)